pkg/auth/providers/github/github_client.go

```diff
@@ -37,27 +36,24 @@ func (g *GClient) getAccessToken(code string, config *v32.GithubConfig) (string,
 
 	b, err := g.postToGithub(url, form)
 	if err != nil {
-		logrus.Errorf("Github getAccessToken: GET url %v received error from github, err: %v", url, err)
-		return "", err
+		return "", fmt.Errorf("github getAccessToken: GET url %v received error from github, err: %v", url, err)
```

Comment:
This error seems confusing? It implies that it's doing a GET just after a call to `postToGithub` which in fact does a POST?

Reply:
Sharp eyes 😄 . That's what the message said before. I'll definitely fix it.